inst/extdata/variable_details.csv

```diff
@@ -4,11 +4,11 @@ active_transport,N/A,cont,"cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs201
+ADL_01,ADL_01_cat2_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p::RACE_6A, cchs2007_2008_p::RAC_6A, cchs2001_i::RACA_6A, cchs2003_i::RACC_6A, cchs2005_i::RACE_6A, [ADL_01]",ICES confirmed,cat,2,2,No,No,N/A,2,No,Help preparing meals,Needs help - preparing meals,,,,active,v3.0.0alpha,
+ADL_01,ADL_01_cat2_NA::a,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p::RACE_6A, cchs2007_2008_p::RAC_6A, cchs2001_i::RACA_6A, cchs2003_i::RACC_6A, cchs2005_i::RACE_6A, [ADL_01]",ICES confirmed,cat,NA::a,2,not applicable,not applicable,N/A,6,not applicable,Help preparing meals,Needs help - preparing meals,,,,active,v3.0.0alpha,
+ADL_01,ADL_01_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p::RACE_6A, cchs2007_2008_p::RAC_6A, cchs2001_i::RACA_6A, cchs2003_i::RACC_6A, cchs2005_i::RACE_6A, [ADL_01]",ICES confirmed,cat,NA::b,2,missing,missing,N/A,"[7,9]",don't know (7); refusal (8); not stated (9),Help preparing meals,Needs help - preparing meals,,,,active,v3.0.0alpha,
+ADL_01,ADL_01_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p::RACE_6A, cchs2007_2008_p::RAC_6A, cchs2001_i::RACA_6A, cchs2003_i::RACC_6A, cchs2005_i::RACE_6A, [ADL_01]",ICES confirmed,cat,NA::b,2,missing,missing,N/A,else,else,Help preparing meals,Needs help - preparing meals,,,,active,v3.0.0alpha,

```

Comment:
@caitlink12 Can you explain why you removed the PUMF 2015-2016 databases from these rows?